### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by ricokahler**
This is the PR for initial release 🎉. It consolidates many of the other PRs.

### CODE REVIEW

File: docs/introduction/what-is-resift.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+id: what-is-resift
+title: What is ReSift?
+sidebar_label: What is ReSift?
+---
+
+Simply put, **ReSift is a state management library for fetches** with the goal of giving your team a capable standard for fetching, storing, and reacting to data.
```
[the remark this answers is not in the document]
Comment (ricokahler) [reply docs/introduction/what-is-resift.md]: I have an explanation for what Relay and Apollo are in the comparison doc. I changed the copy of the link to include the word "definitions".

image

File: docs/introduction/demo-app.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+We recommend you [open the Sandbox](https://codesandbox.io/s/resift-notes-xwp9r) in a new page and play around with it.
+
+---
+
+This is a full demo application including:
+
+- all CRUD actions,
+- updating data across different "namespaces", and
+- carefully placed loading spinners.
```
[the remark this answers is not in the document]
Comment (ricokahler) [reply docs/introduction/demo-app.md]: I just removed the ","s and the "and"

File: docs/introduction/demo-app.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+It's note taking app! Read the pre-made notes to learn more, then edit the code to get better idea of how it works.
+
+<iframe src="https://codesandbox.io/embed/resift-notesj-xwp9r?fontsize=14" title="ReSift Notes" allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>
+<br />
```
[the remark this answers is not in the document]
Comment (ricokahler) [reply docs/introduction/demo-app.md]: love this idea

File: docs/main-concepts/whats-a-fetch.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+ReSift has this concept of a **fetch** (used as a noun). Fetches are tokens you pass to different ReSift functions to get _stuff_ about your fetch.
+
+Here's the general workflow steps:
+
+1. create a **fetch factory** that will produce fetch instance/tokens.
+2. (within a component,) call the fetch factory to get a **fetch instance** (aka just a "fetch").
+3. pass that fetch into ReSift functions to get _stuff_
+
+```js
```
[the remark this answers is not in the document]
Comment (ricokahler) [reply docs/main-concepts/whats-a-fetch.md]: 🤔

File: docs/introduction/what-is-resift.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+id: what-is-resift
+title: What is ReSift?
+sidebar_label: What is ReSift?
+---
+
+> 👋 **Too many words? Head over to the [Quick glance](./quick-glance.md)** 👉
+
+**ReSift is a state management library for fetches** with the goal of giving your team a capable standard for fetching, storing, and reacting to data.
```
[the remark this answers is not in the document]
Comment (ricokahler) [reply docs/introduction/what-is-resift.md]: ✅

File: babel.config.js [hunk 6]
```diff
@@ -11,5 +11,5 @@ module.exports = {
     ],
     '@babel/preset-react',
   ],
-  plugins: ['@babel/plugin-proposal-class-properties'],
+  plugins: ['@babel/plugin-proposal-class-properties', '@babel/plugin-proposal-optional-chaining'],
```
Comment (JulesPatry) [inline babel.config.js]: 👍

**Pull request merged**